src/components/tabs/tabs.test.js

```diff
@@ -617,26 +317,6 @@ context("Testing Tabs component", () => {
       }
     );
 
-    it.each(["error", "warning", "info"])(
-      "should verify when the ValidationIcon is hovered over that a summary of the %s messages is displayed",
-      (validationMessage) => {
-        const validation = { [validationMessage]: validationMessage };
-
-        CypressMountWithProviders(
-          <TabsComponentWithValidationsSummary validation={validation} />
-        );
-
-        tabById(1)
-          .trigger("mouseover")
-          .then(() => {
-            tooltipPreview().should(
-              "have.text",
-              `• ${validationMessage}\n• ${validationMessage}\n• ${validationMessage}`
-            );
-          });
-      }
-    );
```

Comment:
Why is this test removed?

Reply:
Validation prop has been removed, no such text format can be found in the master storybook for Tabs component